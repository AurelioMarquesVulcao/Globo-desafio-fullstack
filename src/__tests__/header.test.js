import React from 'react';
import { shallow } from 'enzyme';
import Header from '../components/header';
import toJson from 'enzyme-to-json';


describe('Testing Header Component', () => {
    it('should render correctly', () => {
        const wrapper = shallow(
            <Header />
        );
        expect(toJson(wrapper)).toMatchSnapshot();

    });
});