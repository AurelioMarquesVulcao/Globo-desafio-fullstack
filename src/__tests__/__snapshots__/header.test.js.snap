// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header Component should render correctly 1`] = `
<div>
  <div
    className="butonsLeft"
  >
    <button
      className="button3"
    />
    <button
      aria-controls="navbarNavAltMarkup"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="/navbarNavAltMarkup"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="alignLeft"
    >
      <a
        className="button2"
        href="/"
      >
        <svg
          height="1em"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 0c-2.995 2.995-7.486 4-11 4 0 8.583 5.068 16.097 11 20 5.932-3.903 11-11.417 11-20-3.514 0-8.005-1.005-11-4zm-7.376 12h7.376v-9.356c2.965 2.238 6.457 3.004 8.912 3.25-.204 2.191-.755 4.236-1.536 6.106h-7.376v9.549c-2.771-2.111-5.643-5.4-7.376-9.549z"
          />
        </svg>
      </a>
      <a
        className="button1"
        href="/"
      >
        <img
          alt="description of IMAGEM"
          className="img"
          src="icons8-person-30.png"
        />
      </a>
      <form
        className="form-inline"
      >
        <input
          aria-describedby="basic-addon1"
          aria-label="Username"
          className="form-control"
          placeholder="Pesquisar..."
          type="text"
        />
        <a
          className="buttonSearch"
          href="/"
        >
          <img
            alt="undefine"
            id="iconsvg"
            src={
              Object {
                "ReactComponent": Object {
                  "$$typeof": Symbol(react.forward_ref),
                  "render": [Function],
                },
                "__esModule": true,
                "default": "svg1.svg",
              }
            }
          />
        </a>
      </form>
    </div>
    <div
      className="exit"
      id="right"
    >
      <img
        alt="undefine"
        id="iconsvg"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "svg2.svg",
          }
        }
      />
    </div>
    <div
      className="options"
      id="right"
    >
      <svg
        className="bi bi-gear-fill"
        fill="currentColor"
        height="1em"
        viewBox="0 0 16 16"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          clipRule="evenodd"
          d="M9.405 1.05c-.413-1.4-2.397-1.4-2.81 0l-.1.34a1.464 1.464 0 01-2.105.872l-.31-.17c-1.283-.698-2.686.705-1.987 1.987l.169.311c.446.82.023 1.841-.872 2.105l-.34.1c-1.4.413-1.4 2.397 0 2.81l.34.1a1.464 1.464 0 01.872 2.105l-.17.31c-.698 1.283.705 2.686 1.987 1.987l.311-.169a1.464 1.464 0 012.105.872l.1.34c.413 1.4 2.397 1.4 2.81 0l.1-.34a1.464 1.464 0 012.105-.872l.31.17c1.283.698 2.686-.705 1.987-1.987l-.169-.311a1.464 1.464 0 01.872-2.105l.34-.1c1.4-.413 1.4-2.397 0-2.81l-.34-.1a1.464 1.464 0 01-.872-2.105l.17-.31c.698-1.283-.705-2.686-1.987-1.987l-.311.169a1.464 1.464 0 01-2.105-.872l-.1-.34zM8 10.93a2.929 2.929 0 100-5.86 2.929 2.929 0 000 5.858z"
          fillRule="evenodd"
        />
      </svg>
    </div>
    <div
      className="home"
      id="right"
    >
      <img
        alt="undefine"
        id="iconsvg"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "svg3.svg",
          }
        }
      />
    </div>
    <div
      className="inputUser"
      id="right"
    >
      <a
        className="inputUser text-white"
        href="/"
      >
        <img
          alt="description of IMAGEM"
          className="img"
          src="icons8-person-30.png"
        />
        INCLUIR USU√ÅRIO
      </a>
    </div>
    <div
      className=" filter button2 "
      data-target="#filterUser"
      data-toggle="modal"
      id="right"
      type="button"
    >
      <img
        alt="undefine"
        id="iconsvg"
        src={
          Object {
            "ReactComponent": Object {
              "$$typeof": Symbol(react.forward_ref),
              "render": [Function],
            },
            "__esModule": true,
            "default": "icon-set2.svg",
          }
        }
      />
    </div>
  </div>
</div>
`;
